src/runtime/policy/tests/policy-test.ts

```diff
@@ -243,6 +243,19 @@ policy MyPolicy {
 }`), 'A definition for child already exists.');
   });
 
+  it('allowed usages defaults to any', async () => {
+    const policy = await parsePolicy(`
+policy MyPolicy {
+  from Abc access {
```

Comment:
could you add a test here for the policy's toString() behavior?

Reply:
Done